docs/messages/0041.md

```diff
@@ -4,42 +4,12 @@ The section "piral" in the "package.json" file is missing.
 
 ## Description
 
-The Piral CLI has to get some meta information for a pilet from its package.json.
-The meta information include its name, version, which Piral instance to use, as well
-as other relevant infos.
-
-Make sure that you modified the package.json correctly using the specification for
-pilets or that the pilet was initially created / scaffolded by the Piral CLI using
-the
-
-```sh
-pilet new
-```
-
-command.
-
-Specifically, the package.json needs to contain a special key called `piral`, which
-contains an object with additional fields.
+The Piral CLI has to get some meta information for a pilet from its package.json.
The meta information include its name, version, which Piral instance to use, as well
as other relevant infos.

Make sure that you modified the package.json correctly using the specification for
pilets or that the pilet was initially created / scaffolded by the Piral CLI using
the

```sh
pilet new
```

command.

Specifically, the package.json needs to contain a special key called `piral`, which
contains an object with additional fields.
```

Comment:
Ah apparently it did have an impact on git.

Can you please revert this and set your global git attributes to only use `\n` and not `\r\n`? The additional `\r` is quite annoying and should not go into the repo. Thanks!

Reply:
Yep I will change it. Sadly these changes are hard to see in vscode working trees :\

Reply:
Agreed.

Reply:
I pushed the updated generated docs (11 Files). Now with the correct line endings